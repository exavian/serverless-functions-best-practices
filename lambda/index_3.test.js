const {handler} = require("./index_3")

describe("Lambda handler function", function () {

  it("should be a function", function () {
    expect(typeof handler).toBe("function")
  })

  it("should match snapshot", function (done) {
    const event = {name: "Quicken Loans"}

    handler(event, {}, (error, response) => {
      expect(error).toBeNull()
      expect(JSON.parse(response)).toMatchSnapshot()
      done()
    })
  })

})
