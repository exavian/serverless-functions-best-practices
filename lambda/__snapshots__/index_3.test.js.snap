// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lambda handler function should match snapshot 1`] = `
Object {
  "name": "Unit testing Lambda locally",
}
`;
